example-app/src/app/components/camera-modal/camera-modal.component.ts

```
@@ -0,0 +1,626 @@
+  public readonly y = input<number>(0);
+  public readonly width = input<number>(0);
+  public readonly height = input<number>(0);
+  public readonly aspectRatio = input<'4:3' | '16:9' | 'fill'>('fill');
+
+  // Picture settings inputs
+  public readonly pictureFormat = input<PictureFormat>('jpeg');
+  public readonly pictureQuality = input<number>(85);
+  public readonly useCustomSize = input<boolean>(false);
+  public readonly pictureWidth = input<number>(1920);
+  public readonly pictureHeight = input<number>(1080);
+  public readonly aspectRatio = input<'4:3' | '16:9'>('4:3');
```

Biome (1.9.4)

[error] Duplicate class member name "aspectRatio"



(lint/suspicious/noDuplicateClassMembers)